Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionContext.h

```diff
@@ -170,6 +170,13 @@ WK_CLASS_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))
  */
 @property (nonatomic, copy) NSString *uniqueIdentifier;
 
+/*!
+ @abstract Determines whether Web Inspector can inspect the @link WKWebView @/link instances for this context.
+ @discussion A context can control multiple `WKWebView` instances, from the background content, to the popover.
+ You should set this to `YES` when needed for debugging purposes. The default value is `NO`.
+*/
+@property (nonatomic, getter=isInspectable) BOOL inspectable;
```

Comment:
Will this property expose itself in Swift as `isInspectable`, or do we need to add `NS_SWIFT_NAME(isInspectable)` to match the `inspectable` property from `JSContext` and `WKWebView`?